completion/git-machete.completion.bash

```diff
@@ -21,6 +21,7 @@ _git_machete() {
   local discover_opts="-C --checked-out-since= -l --list-commits -r --roots= -y --yes"
   local fork_point_opts="--inferred --override-to= --override-to-inferred --override-to-parent --unset-override"
   local github_create_pr_opts="--draft"
+  local github_checkout_pr_opts="PR_number"
```

Comment:
Pls update `github_subcommands` as well! coz as for not `git machete github ch<tab>` doesn't autocomplete as expected

Reply:
Also, wrt. `PR_number`... what's the purpose of this?... AFAICS it'll complete `git machete github checkout-pr P<tab>`... but this doesn't seem very useful (nor an established convention in CLIs) ;)

Reply:
TBH I'd leave an empty `COMPREPLY=('')` after `checkout-pr`... we could fetch the open PR numbers from github but that's way too much engineering FWIW